lib/src/Base/Func/SpecFunc/SpecFunc.cxx

```diff
@@ -274,9 +274,12 @@ Scalar SpecFunc::LargeCaseDeltaLogBesselI10(const Scalar x)
 
 Scalar SpecFunc::DeltaLogBesselI10(const Scalar x)
 {
-  if (x <= 0.0) return LowestScalar;
+  if (!(x > 0.0)) return LowestScalar;
   // Small argument
-  if (std::abs(x) <= 22.0) return std::log(SmallCaseBesselI1(x) / SmallCaseBesselI0(x));
+  if (!(x > 0.848296173838189821792665527416e-2)) return std::log(0.5 * x) + x * x * (-0.125 + 5.0 * x * x / 384.0);
```

Comment:
reference please

Reply:
Maple gives this threshold to have a relative error less than double precision between the exact value and the truncated series. Added a comment in the code.